File: src/AceRental.Application/Clients/Dtos/ClientReservationDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace AceRental.Application.Clients.Dtos
{
    public class ClientReservationDto
    {
        public required string ClientNumber { get; set; }
        public required string FirstName { get; set; }
        public required string LastName { get; set; }
        public required string Email { get; set; }
        public string? PhoneNumber { get; set; }
        public string? Address { get; set; }
    }
}

File: AceRental.FrontEnd/ClientApp/src/app/interfaces/client-reservation-dto.ts
export interface ClientReservationDto {
    ClientNumber: string;
    RaisonSociale?: string;
    FirstName : string;
    LastName : string;
    Email : string;
    Password: string;
    Civilite: string;
    TelNumber?: string;
    PhoneNumber?: string;
    Address?: string;
    ComplementAdresse?: string;
    PostalCode: number;
    City?: string;
    CreateAt: Date;
}
